cuenca_validations/types/queries.py

```diff
@@ -39,6 +39,13 @@ class QueryParams(BaseModel):
 
     class Config:
         extra = Extra.forbid  # raise ValidationError if there are extra fields
+        fields = {
+            'count': {'description': 'Set `true` value to get only a counter'},
+            'page_size': {'description': 'Number of items per page'},
+            'limit': {'description': 'Limit of items to query'},
+            'created_before': {'description': 'Max created date of items'},
+            'created_after': {'description': 'Min created date of items'},
```

Comment:
están mal descritos estos dos

Reply:
Si te digo no vas a aprender, jaja ntc. Es más un tema de ahondar más en la explicación
`Date before the creation of items, this field represent in terms of pagination the max date or the most recent item`

`Date after the creation of items, this field represent in terms of pagination the minimum date or the oldest item`